### PR TITLE
Updating accessible colours into package

### PR DESCRIPTION
**Pull request by TRENDS-Rachel-Avbulimen**
## Proposed changes

I have added the 3 new accessible colour palettes into the dftplotr package, as well as updated the README files to reflect this. 

## Types of changes

What types of changes does your code introduce?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (change which fixes an issue)
- [x] New feature (change which adds functionality)
- [x] Documentation Update (change to naming or other documentation)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR._

- [ ] I have added unit tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)
- [x] I have checked that my changes have not broken any other functionality
- [ ] I have linked to any issues this PR fixes 

## Points for review

Add checkboxes here for any specific aspects of your PR you would like to be checked in peer review. If you don't specify anything here, the reviewer will check:

- [ ] All of your unit tests pass
- [ ] Your code is clear and easy to understand
- [x] You have documented any new features
- [ ] They can run the code you have written
- [x] Your changes do not break any other functionality

## Who is reviewing this PR?

@DIPAD-Fran-Bryden

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -5,15 +5,16 @@ Version: 0.2.1
 Authors@R: 
     c(
     person("Linda", "Bennett", , "linda.bennett@dft.gov.uk", role = c("aut")),
-    person("Francesca", "Bryden", , "francesca.bryden@dft.gov.uk", role = c("aut", "cre")))
```
Comment (DIPAD-Fran-Bryden) [inline DESCRIPTION:8]: Just update the version numbering so we're on 0.3.0 now

File: README.md [hunk 2]
```diff
@@ -124,15 +136,22 @@ common forms of colour blindness, and should also be discernable in
...
 meet WCAG 2.0 accessibility guidance.
 
-Four of the created palettes (`main.palette`, `electric.brights`,
-`just.beachy` and `cycling.hills`) contain four colours each, and meet
-WCAG 2.0 AAA guidance. All colours have a contrast ratio of at least
+Two of the created accessible palettes (`travel.direction`, `country.lane`) 
+contain 8 colours each, and only one accessible palette (`joyful.journey`) 
+contains 6 colours each, and meet WCAG 2.0 AAA guidance. 
+
+Three of the created legacy palettes (`electric.brights`, `just.beachy`
```
Comment (DIPAD-Fran-Bryden) [inline README.md:143]: I would remove mentions of legacy palettes here, just because they're not as good

**Review by DIPAD-Fran-Bryden — APPROVED** [review]
Very neat! Two tiny comments to address, then happy for it to be merged in

File: DESCRIPTION (hunk 1 above)
Comment (TRENDS-Rachel-Avbulimen) [reply DESCRIPTION:8]: This has been updated now @DIPAD-Fran-Bryden

File: README.md (hunk 2 above)
Comment (TRENDS-Rachel-Avbulimen) [reply README.md]: I have removed the paragraph that starts off with "Three of the created legacy palettes..." Did you mean you wanted me to remove all instances of the word 'legacy' from the README files @DIPAD-Fran-Bryden ?

**Pull request merged**